.github/workflows/releaseNigthly.yml

```diff
@@ -236,3 +236,26 @@ jobs:
       run: $GITHUB_WORKSPACE/.github/scripts/upload_failure_logs.sh
       env:
         PLATFORM_TARGET: ${{matrix.target}}
+
+
+  Trigger_Docker:
+    needs: [Linux, Macos]
```

Comment:
Just to be clear, we don't need macOS binaries to be the images.
I suppose you added it so that we only trigger the image is everything passed.

Reply:
It wasn't :)
I will remove it, no need to block the docker if something goes wrong on macos.